### ISSUE TITLE
Add a Transformer Wrapper

### ISSUE THREAD

**Issue by kulbachcedric** [online-ml/river-torch#48]
Use AE as preprocessors within river

**Comment by kulbachcedric** [comment]
I added an Embedding Transformer module. However the tests are currently failing. 
@lucasczz and @hoanganhngo610, the output of the transformer is currently embedding_1, embedding_2 ... embedding_[Embedding_dim]. Do you have any other ideas for the naming. 

And further I think an example would be great. Currently I am working with the SMSSpam Dataset, where I am classifying the the messages on a highly unbalanced dataset. Do you have any other ideas where we could show a use case for the embedding transformer? 

Best
Cedric

**Comment by lucasczz** [comment]
Good idea! I think the feature naming makes sense. Text classification is probably where this kind of embedding works best, so the SMS spam dataset is likely the most suitable of the River datasets to use for the example.
